### DISCUSSION TITLE
Reverification of Academic Affiliation

### DISCUSSION THREAD

**Discussion by SobhanBose** [community/community#61294] [GitHub Education]
### Hi everyone,

I am a student pursuing undergraduate degree. I had previously enrolled into Github Student Developer Pack as a student. When I am trying to upload my college I'd card for reverification, which has the duration of my course clearly mentioned on it, it is being rejected saying I have to provide a school issued email address. The college in which I study does not issue email addresses to students.

I have also tried to type in the name of the college without selecting it from the dropdown, as mentioned in some other discussions, but then it's asking me to select my college on the map. But my college is not present on the map, or more accurately it is wrongly named and hence would not match the name on my college I'd card.

I triend opening a ticket mentioning this, but it was closed instantly with a predefined message which was of no help. I am not able to contact the github support team in anyway.

**Comment by nonvenomous** [comment]
We're in the same boat. Here a bigger thread https://github.com/orgs/community/discussions/59360

**Reply by SobhanBose** [reply]
I have raised a ticket. Let's see how long it takes for them to reply. I did the same for Azure cause its the same problem. Hoping to get a reply soon cause I really want to complete the Microsoft Learn AI challenges.

**Comment by samus-aran** [comment]
Hi @SobhanBose  👋

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

**You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this.** 

Hope this helps! 👍

**Comment by manik-007** [comment]
@SobhanBose 
Hello, i can understand your situation so suggesting you an alternating way to get student developer pack if you are not getting it this way.
you can consider this way but i did not encourage everyone to do this. Probaly you can fill the email address and id card image of one of your friend whose college is listing in the dropdown, and who don't bother much about his/her email. I have seen many of my friends who are not even bothered about their email and they didn't even heard about GitHub, sad but true. So, this way you can get developers pack.

I hope this will helps you.
